src/components/close-button/close-button.jsx

```diff
@@ -7,6 +7,8 @@ import closeIcon from './icon--close.svg';
 import closeIconOrange from './icon--close-orange.svg';
 import backIcon from '../../lib/assets/icon--back.svg';
 
+let closeIcons = {};
```

Comment:
Need to initialize this variable so it can be used in the code below, even though it's not actually set until lower down